src/resources/views/show.blade.php

```diff
@@ -3,7 +3,7 @@
 @section('content-header')
 	<section class="content-header">
 	  <h1>
-	    {{ trans('backpack::crud.preview') }} <span>{{ $crud->entity_name }}</span>
+	    {{ trans('backpack::crud.preview') }} <span class="text-lowercase">{{ $crud->entity_name }}</span>
```

Comment:
Why do we have `text-lowercase` in that span? German, for one, has rules about capitalising nouns for example?